R/deployApp.R

```diff
@@ -189,7 +189,8 @@ deployApp <- function(appDir = getwd(),
 
   if (upload) {
     # create, and upload the bundle
-    withStatus("Uploading application bundle", {
+    withStatus(paste("Uploading bundle for application:",
+                     application$id) {
```

Comment:
The paren is closed later (`withStatus` takes an expression). The colon is there. What am I missing?